server/tserver/src/main/java/org/apache/accumulo/tserver/session/ScanSession.java

```diff
@@ -177,4 +179,12 @@ public void finishedRun(long start, long finish) {
   public TabletResolver getTabletResolver() {
     return tabletResolver;
   }
+
+  @Override
+  public boolean cleanup() {
+    super.cleanup();
```

Comment:
should probably check the return value if calling it

Reply:
I'll modify and merge. Thx!